plutus-metatheory/src/Main.lagda

```diff
@@ -66,13 +66,13 @@ instance
 
 postulate
   ByteString : Set
-  getContents : IO ByteString
-  readFile : String → IO ByteString
+  getContents : IO String
```

Comment:
I wonder if we want to use `Data.Text.IO`'s `getContents` and `readFile` like [here](https://github.com/input-output-hk/plutus/pull/4513/files#diff-e0a808fa4cee3ad0e779b23cbd599d33c8d316f425e6d38a62910ddc1c483c53R270-R272)?